lib/SVF-FE/LLVMModule.cpp

```diff
@@ -227,18 +228,79 @@ void LLVMModuleSet::initialize()
     }
 }
 
+std::vector<llvm::Function *> LLVMModuleSet::getLLVMGlobalFunctions(
+        llvm::GlobalVariable *global)
+{
+    class LLVMGlobalFunction {
+    // llvm.global_ctors and llvm.global_dtors
+    public:
+        unsigned priority;
+        llvm::Function *func;
+        LLVMGlobalFunction(){};
+        LLVMGlobalFunction(unsigned _priority, llvm::Function *_func)
+              : priority(_priority), func(_func){};
+        bool operator>(const LLVMGlobalFunction &other) const
+        {
+            return priority > other.priority;
+        }
+    };
+
+    std::priority_queue<LLVMGlobalFunction, std::vector<LLVMGlobalFunction>,
+                        greater<LLVMGlobalFunction>>
+            queue;
+    std::vector<llvm::Function *> result;
+
+    llvm::ConstantArray *globalFuncArray =
+            llvm::dyn_cast<llvm::ConstantArray>(global->getInitializer());
+    if (globalFuncArray) {
+        for (unsigned int i = 0; i < globalFuncArray->getNumOperands(); ++i) {
+            llvm::ConstantStruct *globalFuncItem =
+                    llvm::dyn_cast<llvm::ConstantStruct>(
+                            globalFuncArray->getOperand(i));
```

Comment:
could you use SVF::dyn_cast and put this dyn_cast into the `if` branch?

if(ConstantStruct *globalFuncItem = SVF::dyn_cast<llvm::ConstantStruct>(globalFuncArray->getOperand(I))){
...
}